syncd/VendorSai.cpp

```diff
@@ -998,6 +998,60 @@ sai_status_t VendorSai::bulkCreate(
             object_statuses);
 }
 
+sai_status_t VendorSai::bulkCreate(
```

Comment:
@kcudnik this change be added as another PR as this is NOT relevant as per the PR title

Reply:
read this comment: https://github.com/sonic-net/sonic-sairedis/pull/1430#issuecomment-2405155952, this is 1st pr: https://github.com/sonic-net/sonic-sairedis/pull/1431